import-brage/src/test/java/no/unit/nva/importbrage/XmlImportTest.java

```diff
@@ -1,36 +1,26 @@
 package no.unit.nva.importbrage;
 
 
+import no.unit.nva.importbrage.metamodel.BrageContributor;
 import org.junit.jupiter.api.Test;
 
 import java.io.File;
 import java.io.IOException;
-import java.util.Set;
 
-import static java.util.Objects.isNull;
 import static org.junit.jupiter.api.Assertions.assertNotNull;
-import static org.junit.jupiter.api.Assertions.assertTrue;
 
 class XmlImportTest {
 
     public static final String OVERCOMPLETE_EXAMPLE = "/overcomplete_example.xml";
-    public static final Set<String> LANGUAGE_VALUES = Set.of("*", "en", "en_US");
 
     @Test
     void xmlImportLoadsData() throws IOException {
```

Comment:
One test is xmlImportReturnsAListOfContributorsWhenInputIsFileContainingContributors and another test could be 
xmlImportDoesNotCreateContributorsFromElementsThatAreNotContributors